File: Card.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ConsoleApplication2
{
    public class Card
    {
        private int val;

        public Card(int v)
        {
            val = v;
        }

        public int getRank()
        {
            return val % 13;
        }

        public int getSuite()
        {
            return val / 13;
        }

        public int Val
        {
            get { return val; }
        }
    }
}

File: other_cg/libTypes/Card.java
package libTypes;

public class Card {
    private int val;
    public Card(int v){
     val =v;  
    }
    
    public int getRank(){
       return val % 13;      
    }
    
    public int getSuite(){
       return val / 13;      
    }
  
    public int getVal(){
      return val ;      
   }
    
    
    
}
